SEQLinkage/Main.py

```diff
@@ -152,9 +149,11 @@ def checkParams(args):
     return True
 
 # Cell
-def main(args):
+def main():
     '''the main encoder function'''
+    args = Args().get()
     checkParams(args)
+    print(env.version,'env')
```

Comment:
Please don't print anything

Reply:
This line is a test line, which will be removed.